service/src/test/java/bio/terra/workspace/common/GcpCloudUtils.java

```diff
@@ -77,15 +79,29 @@ public void populateBqTable(GoogleCredentials userCredential, String projectId,
     logger.debug("Employee Table: {}", createdEmployeeTable);
 
     // Add row to table
-    // Don't call insertAll() with InsertAllRequest. That inserts via stream. Stream buffer may not
-    // be copied for up to 90 minutes:
-    // https://cloud.google.com/bigquery/docs/streaming-data-into-bigquery#dataavailability
-    // Instead, use DDL to insert rows.
-    bigQueryClient.query(
-        QueryJobConfiguration.newBuilder(
-                "INSERT INTO `%s.%s.%s` (employee_id) VALUES(%s)"
-                    .formatted(projectId, datasetId, BQ_EMPLOYEE_TABLE_NAME, BQ_EMPLOYEE_ID))
-            .build());
+    // Retry because if project was created recently, it takes time for bigquery.jobs.create to
+    // propagate
+    for (int i = 0; i < 10; i++) {
```

Comment:
nit: small preference for putting the retry values into variables so they are easier to modify.